### ISSUE TITLE
Add Alternative Texts

### ISSUE THREAD

**Issue by Rakielle** [uyuni-project/uyuni-project.github.io#92]
# Details

As part of [openSUSE/mentoring#205](https://github.com/openSUSE/mentoring/issues/205), this issue aims to improve the accessibility of Uyuni website, by addressing issues dentified by WAVE.

1. **Logo Alt Text:**
The current text logo has no alt text. This is an accessibility issue because the content and purpose of the image will not be available to screen reader users or when the image is unavailable.
Adding an alt text will help to convey the content to people who use assistive technologies.</br>
<img src="https://github.com/uyuni-project/uyuni-project.github.io/assets/48498778/ecf24162-2a7d-4158-80ec-c6f207170f76" alt="img2" width="300"></br>

2. **Plus Icon Alt Text:**
The plus icons have no alt text. Since it signifies expandable sections, adding an alt text would help users who rely on screen readers understand what it's for.</br>
<img src="https://github.com/uyuni-project/uyuni-project.github.io/assets/48498778/c7101a20-4e32-4853-9afe-efcce5507023" alt="img1" width="350">
</br>

# Reference
[Non-text content](https://webaim.org/standards/wcag/checklist#sc1.1.1)

**Comment by Sauradip07** [comment]
Please assign this issue to me
